bionic/deriver.py

```diff
@@ -218,6 +213,7 @@ def _populate_entity_info(self, entity_name):
         self._task_lists_by_entity_name[entity_name] = provider.get_tasks(
             dep_key_spaces_by_name,
             dep_task_key_lists_by_name)
+        self._docstring_by_entity_name[entity_name] = provider.attrs.docstring
```

Comment:
I merged a PR recently that changed the single `provider.attrs.docstring` to multiple `provider.attrs.docs`, so you'll probably want to use something like `provider.doc_for_name(entity_name)` here now.

(I also changed most internal uses of `docstring` to `doc`, to make it clearer whether we're referring to a Bionic entity documentation string (`doc`), or an actual Python function docstring (`docstring`).)

Reply:
Saw this when I rebased on master, and will make the changes- thanks!